pkg/apis/sources/v1alpha1/ibmmq_lifecycle.go

```diff
@@ -0,0 +1,76 @@
+/*
+Copyright 2021 TriggerMesh Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	"fmt"
+	"strings"
+
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"knative.dev/pkg/apis"
+	duckv1 "knative.dev/pkg/apis/duck/v1"
+)
+
+// Managed event types
+const (
+	IBMMQSourceEventType = "io.triggermesh.ibmmqsource"
```

Comment:
Really not a fan of this event type to be honest @tzununbekov . how about? 
```suggestion
	IBMMQSourceEventType = "io.triggermesh.ibm.mq.message"
```